testsuite/features/secondary/min_salt_formulas.feature

```diff
@@ -160,8 +162,10 @@ Feature: Use salt formulas
      And the keymap on "sle_minion" should be "us.map.gz"
      And the language on "sle_minion" should be "en_US.UTF-8"
 
-  Scenario: Cleanup: uninstall formula package from the server
+  Scenario: Cleanup: uninstall formula package from the server and disable repositories on the minion
```

Comment:
no need to put it in scenario title, it's an implementation detail IMHO

Reply:
Removed as well